pt_PT/LC_MESSAGES/inventory.po

```diff
@@ -4,31 +4,31 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 
 msgid "DECOR"
-msgstr "Decor"
+msgstr "Decor."
```

Comment:
I don't think a period is necessary here.

Reply:
it's word shortening, just in case it's displayed on a DS resolution